src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -430,20 +434,22 @@ async def set_tpz_io_settings(self, voltage_limit, hub_analog_input):
         else:
             raise ValueError("voltage_limit must be 75 V, 100 V or 150 V")
 
-        payload = st.pack("<HHHHH", 1, voltage_limit, hub_analog_input, 0, 0)
+        payload = st.pack(
+            "<HHHHH", _CHANNEL, voltage_limit, hub_analog_input, _RESERVED, _RESERVED
+        )
         await self.send(Message(MGMSG.PZ_SET_TPZ_IOSETTINGS, data=payload))
 
-    async def get_tpz_io_settings(self):
+    async def get_tpz_io_settings(self) -> tuple[int, int]:
         """Get various I/O settings.
 
         :return: Returns a tuple whose elements are the voltage limit and the
-            Hub analog input. Refer to :py:meth:`set_tpz_io_settings()
+            Hub analog input. Refer to :py:meth:`set_tpz_io_settings()`
             <Tpz.set_tpz_io_settings>` for
             the meaning of those parameters.
         :rtype: a 2 elements tuple (int, int)
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_TPZ_IOSETTINGS, [MGMSG.PZ_GET_TPZ_IOSETTINGS], 1
+            MGMSG.PZ_REQ_TPZ_IOSETTINGS, [MGMSG.PZ_GET_TPZ_IOSETTINGS], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_TPZ_IOSETTINGS, [MGMSG.PZ_GET_TPZ_IOSETTINGS], Tpz._CHANNEL
```
Use class constants.